contracts/programs/store/src/lib.rs

```diff
@@ -192,7 +192,7 @@ pub mod store {
     }
 
     #[access_control(store_owner(&ctx.accounts.store, &ctx.accounts.authority))]
-    pub fn transfer_store_ownership(
+    pub fn transfer_ownership(
```

Comment:
Note: changed `transfer_store_ownership` and `accept_store_ownership` to `transfer_ownership` and `accept_ownership` which better matches the generic `<PROGRAM TYPE>:<transfer / accept>_ownership`

Reply:
Hmm, `transfer_store_ownership` expresses which account is used a bit better. `Store` account is a lot less important now and won't be used for most operations, perhaps it should be renamed to `Owner`/`Authority` or something similar

Reply:
I suggested this change on the basis that any contract that has an `owner` should have the two `transfer/accept_ownership` methods.

Here, `store` contract has an `owner` but it had inconsistently defined `transfer/accept_store_ownership` methods.